src/languages/en-US/commands/misc.json

```
@@ -246,6 +247,29 @@
 			"GloriousGe0rge"
 		]
 	},
+	"resetBirthdayDescription": "Reset your birthday.",
+	"resetBirthdayNotSet": "{{REDCROSS}} You have not set your birthday in this server.",
+	"resetBirthdaySuccess": "{{GREENTICK}} I will not wish you in this server anymore.",
+	"setBirthdayDescription": "Set your birthday to be automatically notified.",
+	"setBirthdayExtended": {
+		"usages": ["Date"],
+		"extendedHelp": "This command needs the settings `birthday.role` or `birthday.channel` to be configured. On a user's birthday, I will give them the birthday role, and send `birthday.message` to the birthday channel.",
+		"explainedUsage": [
+			[
+				"BirthDate",
+				"Your birth-date, in the format YYYY/MM/DD."
+			]
+		],
+		"examples": [
+			"2002/01/01"
+		],
+		"reminder": "By using this command, you consent to Skyra saving your date of birth."
+	},
+	"setBirthdayNotConfigured": "A birthday channel or role has not been configured in this server. Ask an admin to configure one using the conf command!",
```

```suggestion
	"setBirthdayNotConfigured": "{{REDCROSS}} A birthday channel and/or role has not been configured in this server. Ask an admin to configure at least one using either the `{{prefix}}conf` command or through [the dashboard](https://skyra.pw)",
```

Same deal as before with passing in prefix.